card-data/01-character-upbringing.yaml

```
@@ -0,0 +1,264 @@
+- template: &template
+    tags:
+      - character
+      - upbringing
+- template: &tfront
+    <<: *template
+    rule: Flip to get a hit.
+- template: &tback
+    <<: *template
+    rule: Flip when a prompt is narrated by anyone.
+- front: &City
+    <<: *tfront
+    name: City
+    desc: >
+      //"Welcome to the Labri Market! You can find anything here." -- Basler
+
+      You grew up in a highly populated center of trade and culture.
+      People with needs offered what they had, or sometimes more,
+      and you did it too.
```

```suggestion
      You grew up in a highly populated center of trade and culture.
```
I'm not at all sure what you were trying to convey with the removed sentence. But everything works well without it.